scvi/inference/trainer.py

```diff
@@ -225,23 +226,26 @@ def __setattr__(self, name, value):
         else:
             object.__setattr__(self, name, value)
 
-    def train_test(self, model=None, gene_dataset=None, train_size=0.1, test_size=None, seed=0, type_class=Posterior):
-        """
+    def train_test_validate(self, model=None, gene_dataset=None, train_size=0.1, test_size=None, type_class=Posterior):
+        """Creates posteriors ``train_set``, ``test_set``, ``validate_set``. If ``train_size + test_size < 1`` then ``dev_set`` is non-empty.
+
         :param train_size: float, int, or None (default is 0.1)
         :param test_size: float, int, or None (default is None)
         """
         model = self.model if model is None and hasattr(self, "model") else model
         gene_dataset = self.gene_dataset if gene_dataset is None and hasattr(self, "model") else gene_dataset
         n = len(gene_dataset)
         n_train, n_test = _validate_shuffle_split(n, test_size, train_size)
-        np.random.seed(seed=seed)
+        np.random.seed(seed=self.seed)
```

Comment:
Hey @adamgayoso can you change this `seed` here. It has an impact on all the code. You can use: 
`random_state = np.RandomState(seed=self.seed)` and then `permutation = random_state.permutation(n)`